src/Transports.AspNetCore/Errors/RequestError.cs

```diff
@@ -0,0 +1,35 @@
+#nullable enable
+
+using GraphQL.Execution;
+
+namespace GraphQL.Server.Transports.AspNetCore.Errors;
+
+/// <summary>
+/// Represents an error that occurred prior to the execution of the request.
+/// </summary>
+public class RequestError : ExecutionError
```

Comment:
@Shane32 I suppose that after discussion around Apollo metrics and APQ some of those errors (this one + QueryMissingError) may be moved into mainn repo.

Reply:
I think that would be best.  I would merge it in here at least temporarily so we can continue.  When it is added to the main repo, we can remove it here; no big deal.

Reply:
I reviewed APQ PR completely and I'm sure it will be merged tomorrow. A lot of household chores this week. I'll continue to review this one PR now.